internal/project/js/toolchain.go

```diff
@@ -147,7 +147,7 @@ func (toolchain *Toolchain) CompileDockerfile(output *dockerfile.Output) error {
 		Step(step.Copy(filepath.Join(toolchain.sourceDir, "package-lock.json"), "./")).
 		Step(step.Script("npm version ${VERSION}").
 			MountCache(toolchain.meta.NpmCachePath)).
-		Step(step.Script("npm install").
+		Step(step.Script("npm ci && npm cache clean --force").
```

Comment:
if we clean cache after each execution, doesn't that mean `node_modules` is not cached by buildx?

Reply:
I guess it still caches the node modules in the right way.
But it's actually not required to clean that up, as we don't create the image from the same container.
Will fix, thanks.